website/docs/plugin/which-sdk.mdx

```diff
@@ -45,17 +45,13 @@ should consider using the framework if you would benefit from the ability to:
 * Return warning or error diagnostics during resource destruction.
 * Read or write managed resource private state data.
 
-## Do You Need Any Features That the Framework Doesn't Provide (Yet)?
+## Do You Need Any Features That the Framework Does Not Provide (Yet)?
 
 The framework is still under development, and so it may not yet support some
-features that are available in SDKv2. These include the ability to:
+features that are available in SDKv2. In that case, SDKv2 may be a
+better choice.
```

Comment:
I think we can drop this whole section, since there is nothing it cannot do (theoretically 😉 ).

Reply:
Have removed this section.